### ISSUE TITLE
Encoding corruption in Wikiplantbase #Italia

### ISSUE THREAD

**Issue by MattBlissett** [gbif/portal-feedback#3538]
The ABCD archive contains corrupt UTF-8 content, for example in UnitID 69557:

`<abcd:TitleCitation>Lastrucci L., Gonnelli V., Foggi B., 2005. Flora e vegetazione di alcune aree umide dell'altopiano della "Pianca"\235 nell'alta Val Marecchia (provincia di Arezzo, Toscana). Inform. Bot. Ital., 36 (2) (2004): 429-442.</abcd:TitleCitation>`

The \235 is my editor showing bytes which are invalid UTF-8.

(There are probably more cases.)

**Comment by jlegind** [comment]
Forwarded issue to Jörg and Gianni.

**Comment by jlegind** [comment]
A reminder was sent.

**Comment by gibedini** [comment]
Thank you Jan, but I cannot see the offending character. So I just removed the space between "Pianca" and "nell" and inserted a new space manually (space bar). Then I recreated the ABCD archive and these are the last lines of the log:
...
INFO  01-09 17:41:02,347 - Done for all datasets. 93,641 out of 93,641 records archived (0 dropped). 
INFO  01-09 17:41:02,347 - Removing old archives and moving new files to download folder... 
INFO  01-09 17:41:02,349 - Updating inventory... 
INFO  01-09 17:41:02,349 - SUCCESS: Archiving process finished. 
... processing ended. 

If my memory serves me well, a couple of years ago the archiving process would stop when it came to an offending character, so I enforced stricter rules for deleting them before saving records to the database and the problem disappeared... so far! Now, if the log says that all records have been archived with no drop-outs, does it mean that the offending character has been removed and all is neat and safe?
